docs/reference/contributing/guidelines/workflow.md

```diff
@@ -79,6 +87,43 @@ Each feature has a tech lead. This person is responsible for:
 - Rebasing often to track master development.
 - Reviewing any addition to the feature branch (approval required by the feature tech lead or another assigned person).
 
+##### Pull request types
+
+We consider the following pull request types.
+
+##### Fix
+
+A bug fix is backward-compatible internal change that fixes incorrect behavior. The fix provides a description what and how it fixes the issue. Tests are included to catch the issue in the future and confirms the correct behavior.
```

Comment:
"is backward-compatible" to "is a backwards-compatible"

Reply:
"The fix provides a description what and how it fixes the issue."

I'm not sure what's trying to be said here.

Reply:
> I'm not sure what's trying to be said here.

I'll remove that one, it is in git guidance above

> "is backward-compatible" to "is a backwards-compatible"

@AnotherButler I believe both are correct, aren't they? If not, let me know and I'll do edits